### PR TITLE
Set up linters 

### PR DESCRIPTION
**Pull request by DeograciasGraole**
I was making a set up of linters for the project2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+      </div>
+
+      <div class="section-work-Card">
+        <div
+          class="section-work-Card-left"
+          style="background-image: url(Assets/4335772.jpg)"
+        ></div>
+        <!-- <img src="Assets/mobile.jpg" class="" alt="" /> -->
+        <div class="section-work-Card-right">
+          <h1 class="Card-right-title">Facebook 360</h1>
```
Comment (Pamphilemkp) [inline index.html:132]: for accessibility purpose and SEO it is allow to only have one heading title per page, every document should have at most 1 h1 and the remaining can be h22, h3, h4, h5, h6. H1 is used for big titles on a page for example the main title. consider using the remaing heading.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+      </div>
+
+      <div class="section-work-Card">
+        <div
+          class="section-work-Card-left order"
+          style="background-image: url(Assets/imgfood.jpg)"
+        ></div>
+        <!-- <img src="Assets/mobile.jpg" class="" alt="" /> -->
+        <div class="section-work-Card-right">
+          <h1 class="Card-right-title">Tonic</h1>
```
Comment (Pamphilemkp) [inline index.html:164]: Please do this to the remaining h1 on the page, replace them with other headings.   for accessibility purpose and SEO it is allow to only have one heading title per page, every document should have at most 1 h1 and the remaining can be h22, h3, h4, h5, h6. H1 is used for big titles on a page for example the main title. consider using the remaing heading.

**Review by Pamphilemkp — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @DeograciasGraole , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!




## Highlights :green_circle: 


- PR is descriptive and professional :+1: 
- No Linter error :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy] have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can contact us via._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>Graole web</title>
+  </head>
+  <body>
+    <!-- *************************************************************Start header********************************************************-->
```
Comment (basitali111) [inline index.html:11]: Please avoid excessive comments in your code. Over-commenting can clutter your code and make it harder to maintain. Instead, focus on writing clear, self-explanatory code with descriptive names for variables and functions.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+            </nav>
+            <nav>
+              <img class="logo-html" src="Assets/html-5.svg" alt="" />
+              <p>HTML</p>
+            </nav>
+            <nav>
+              <img class="logo-css" src="Assets/css-3.svg" alt="" />
+              <p>Css</p>
+            </nav>
+          </div> -->
```
Comment (basitali111) [inline index.html:315]: Please avoid excessive comments in your code. Over-commenting can clutter your code and make it harder to maintain. Instead, focus on writing clear, self-explanatory code with descriptive names for variables and functions.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+              rows="10"
+              cols="100"
+              name=""
+              id=""
+            ></textarea>
+            <a href="#" class="card-right-button">Get in touch</a>
+          </form>
+        </div>
+      </div>
+    </section> -->
```
Comment (basitali111) [inline index.html:344]: - Remove these comments also.

**Review by basitali111 — APPROVED** [review]
# Approve 🟢 🟢 🟢

Hi 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉
- Kindly Check comment under the pull request make sure to implement it to get approval next time happy coding


giphy

Highlights
No linter errors ✅
Good Pr title and description 💪🏿
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Kindly make use of the provided README.md template provided in your dashboard otherwise the next project will be marked as invalid
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @basitali111 

_As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion by contacting us.

**Pull request merged**